code/__HELPERS/game.dm

```diff
@@ -305,7 +305,6 @@ var/list/DummyCache = list()
 	D.flags=initial(D.flags)
 	D.pass_flags=initial(D.pass_flags)
 	if(pass_flags&PASSTABLE)
-		D.flags      |= TABLEPASS
```

Comment:
This line is actually pretty pointless, since TABLEPASS is never checked anywhere.